src/OrbitGl/PresetsDataView.h

```diff
@@ -68,6 +69,9 @@ class PresetsDataView : public DataView {
 
   static const std::string kMenuActionLoad;
   static const std::string kMenuActionDelete;
+
+ private:
+  const orbit_metrics_uploader::MetricsUploader* metrics_uploader_;
```

Comment:
Looks like all DataViews need metrics_uploader_ anyway. Should we add it to the base class then?

Reply:
For now not all DataViews need it, so I would rather not add it to the base class. But I also don't feel strongly about it and can add it, no problem. wdyt?

Reply:
Could be done when all the DataViews have it, so not in this PR